store/sqlstore/thread_store.go

```diff
@@ -188,3 +188,47 @@ func (s *SqlThreadStore) CreateMembershipIfNeeded(userId, postId string) error {
 	})
 	return err
 }
+
+func (s *SqlThreadStore) UpdateUnreadsByChannel(userId string, channelLastUnreads map[string]int64) error {
```

Comment:
Because this is olny used internally, there is not reason to expose it publicly. I would convert it into a private method.

Reply:
@jespino - 'internally' is subjective :) It's being used by App methods, so I do need to export the method...

Reply:
I don't see any call done from app layer in this pr, looks like this is call from other store methods, so it doesn't need to be directly exposed, or I'm missing something?

Reply:
@jespino  :man_facepalming: you are correct. will fix

Reply:
@jespino  Nope, you managed to confuse me. :)
You cannot make this method private, since it's being called from another Store class by it's interface.
And if the method is private, it cannot be used like:
`	go s.Thread().updateUnreadsByChannel(userId, map[string]int64{channelId: now}) `

Reply:
We discuss this through videoconference and looks like the current way we define the SqlStore doesn't allow this kind of private methods, so we need to address that later in another ticket. For now this is fine.